rosdep/python.yaml

```diff
@@ -2090,6 +2090,9 @@ python-impacket:
   debian: [python-impacket]
   fedora: [python-impacket]
   ubuntu: [python-impacket]
+python-influxdb:
+  debian: [python-influxdb]
+  ubuntu: [python-influxdb]
```

Comment:
This is only available on Ubuntu up until [disco](https://packages.ubuntu.com/search?keywords=python-influxdb&searchon=names&suite=all&section=all); for eoan and focal, I assume it got purged as the general purging of Python 2 packages happened.  So I think we should make specific rule for it for the currently supported releases (xenial and bionic)

Reply:
I've just updated to list xenial, bionic and disco as python-influxdb targets.